test/integration/versioned-api/versioned_api.spec.test.js

```diff
@@ -5,15 +5,6 @@ const { loadSpecTests } = require('../../spec/');
 const { runUnifiedTest } = require('../../tools/unified-spec-runner/runner');
 
 describe('Versioned API', function () {
-  it('should throw an error if serverApi version is provided via the uri', {
-    metadata: { topology: 'single' },
-    test: function () {
-      expect(() => this.configuration.newClient({ serverApi: '1' })).to.throw(
-        /URI cannot contain `serverApi`, it can only be passed to the client/
-      );
-    }
-  });
-
   for (const versionedApiTest of loadSpecTests('versioned-api')) {
```

Comment:
We have a helper function that runs a unified spec test suite:

```typescript
runUnifiedSuite(loadSpecTests('versioned-api'))
```

Reply:
good call, tbh I hadn't even looked at the line below the test that I removed

Reply:
Yeah this is actually something Neal pointed out to me last week 😄 We have the function but we're only using it in a few places

Reply:
I believe when we first merged the unified runner I hadn't made the `runUnifiedSuite` helper so a few copy pastas of the `for` loop made it around we should clean up when we come across them. Thanks!